### ISSUE TITLE
The baseline model presented in the paper

### ISSUE THREAD

**Issue by yug125lk** [YuYin1/DA-GAN#16]
Hi, thank you for sharing this great work. I just wanted to ask you about the results of the baseline model presented in the paper. I experimented with the same datasets (CAS-PEAL R1, Multi PIE) using a baseline model, but I obtained poor results. Did you use a simple pix2pix GAN? Did you change any parameters or loss functions in the baseline model? Could you tell me what the values of the batch size, epochs, learning rate, and summary model are? and thank you.

**Comment by YuYin1** [comment]
We didn't use pix2pix GAN. All the hyper-parameters and loss functions of our baseline are the same as listed in the "option.py" file.
